src/js/tech/html5.js

```diff
@@ -86,6 +87,12 @@ class Html5 extends Tech {
    * @method dispose
    */
   dispose() {
+    let tt = this.el().textTracks;
+
+    tt.removeEventListener('change', Fn.bind(this, this.handleTextTrackChange));
```

Comment:
If `el().textTracks` is a native list then I don't know if the Fn.bind is going to do what you want here. The remove-by-same-id only works with our internal event listeners. The native removeEventListener still looks for the same function.